aws/requests.yaml

```diff
@@ -16,6 +16,11 @@ releases:
         - name: coredns
           version: ">= 1.4.0"
           issue: https://github.com/giantswarm/giantswarm/issues/13771
+    - name: "> 14.1.1 > 13.1.1"
```

Comment:
Does the order of version matter?
`"> 14.1.1 > 13.1.1"` vs `"> 13.1.1 > 14.1.1"`

Reply:
No idea. Who could know? For funsies lets reverse this and push again to see what CI thinks

Reply:
```suggestion
    - name: "> 13.1.1 > 14.1.1"
```

Reply:
Not sure I only saw it some lines below the order was different 🙂 .

Reply:
Its parsed by this function https://github.com/giantswarm/releases/blob/master/releases_test.go#L198 (called https://github.com/giantswarm/releases/blob/master/releases_test.go#L103)